walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,34 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
+
+    String greeting = greeter.greet("   Ada   ");
+
+    // Whitespace should be trimmed
+    Assert.assertEquals("Hello Ada", greeting);
```

Comment:
Google unit-test practices recommend to use the Truth library (https://truth.dev) rather than JUnit's internal assetions.
Truth assertions are more "fluent" and provide better error messages.